examples/bert/README.md

```diff
@@ -10,6 +10,16 @@ In sum, this example showcases:
 * Building and fine-tuning on downstream tasks
 * Use of Texar `TFRecordData` module for data loading and processing
```

Comment:
No `TFRecord*`

Reply:
For `bert` example, I believe there are more TFrecord related issues. I am currently refactoring the `pretrain` module and will polish this example later.